136-stacked-area-chart.Rmd

```diff
@@ -30,7 +30,7 @@ knitr::opts_chunk$set( warning=FALSE, message=FALSE)
 
 
 
-# Most basic stacked area with ggplot2
```

Comment:
I like it when examples are _really_ simple at the beginning of a tutorial. Does _Most basic_ sound weird here? 🤔 (I'm not native)

Reply:
I agree that simple examples are great, and simple heading are also great :) The reader expects the first example to be simple and we can let them dive straight into the code.